src/main/java/org/apache/sling/resourceresolver/impl/mapping/MapEntries.java

```diff
@@ -242,7 +260,16 @@ protected boolean doInit() {
                 try {
                     final Map<String, Map<String, Collection<String>>> loadedMap = this.loadAliases(resolver);
                     this.aliasMapsMap = loadedMap;
-    
+
+                    // warn if there are more than a few defunct aliases
+                    if (conflictingAliases.size() >= MAX_REPORT_DEFUNCT_ALIASES) {
```

Comment:
I see it right below, elements are only added to conflictingAliases while ``conflicting < MAX_REPORT_DEFUNCT_ALIASES``. that means ``conflictingAliases.size()`` is always less than ``MAX_REPORT_DEFUNCT_ALIASES``. 


I would always log conflicts, even if we have only 5.

Reply:
I see,